docs/docsource/Reference/bucket_lifecycle_operations/put_bucket_lifecycle.rst

```diff
@@ -360,13 +360,11 @@ configuration:
    | Transition            | transition rule that  | action is present in  |
    |                       | describes when        | the Rule.             |
    |                       | noncurrent objects    |                       |
-   |                       | transition to the     |                       |
-   |                       | STANDARD_IA or        |                       |
-   |                       | GLACIER storage       |                       |
-   |                       | class.                |                       |
+   |                       | transition to another |			   |
+   |			   | storage class 	   |			   |
```

Comment:
not sure if this is just cos of the view in Github, but there might be a small misalignment in the table here

Reply:
The source looks clean. I think this is an artifact.